packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -26,16 +26,16 @@ dependencies:
     sdk: flutter
   package_info_plus_platform_interface: ^1.0.2
   package_info_plus_linux: ^1.0.5
-  package_info_plus_macos: ^1.2.0
-  package_info_plus_windows: ^1.0.5
+  package_info_plus_macos: ^1.3.0
+  package_info_plus_windows: ^2.0.0
```

Comment:
Is it ok to change here the version to the version in the pull request?

Reply:
Yes, it is fine.